### PR TITLE
kdump: Fix kexec-tools installation kexec-boot test

### PR DESCRIPTION
**Pull request by emmawu**
- Installing/Upgrading kexec-tools in kexec-boot if Fedora
- Rename PrepareKdump() to SetupKdump()
- Move installing/upgrading kexec-tools pkgs from SetupKdump to PrepareKdump

Signed-off-by: xiawu <xiawu@redhat.com>

### CODE REVIEW

**Comment by emmawu** [comment]
Verified on RHEL and Fedora.

**Comment by emmawu** [comment]
Fixed spelling errors...

File: kdump/include/runtest.sh [hunk 1]
```diff
@@ -84,8 +84,8 @@ K_VMCOREPATH=${K_VMCOREPATH:-"/var/crash"}
 [ "${K_ARCH}" = "ia64" ] && K_BOOT="/boot/efi/efi/redhat" || K_BOOT="/boot"
 
 # back up kdump config files
-[ -f "${KDUMP_CONFIG}.bk" ] || cp "${KDUMP_CONFIG}" "${KDUMP_CONFIG}.bk"
-[ -f "${KDUMP_SYS_CONFIG}.bk" ] || cp "${KDUMP_SYS_CONFIG}" "${KDUMP_SYS_CONFIG}.bk"
+[ -f $"${KDUMP_CONFIG}" -a ! -f "${KDUMP_CONFIG}.bk" ] && cp "${KDUMP_CONFIG}" "${KDUMP_CONFIG}.bk"
+[ -f $"${KDUMP_SYS_CONFIG}" -a ! -f "${KDUMP_SYS_CONFIG}.bk" ] && cp "${KDUMP_SYS_CONFIG}" "${KDUMP_SYS_CONFIG}.bk"
```
Comment (emanchado) [inline kdump/include/runtest.sh:88]: Ooh, I had never seen that `$"${VAR}"`, what does it do?

Comment (emmawu) [reply kdump/include/runtest.sh:88]: gosh..it's an error. Thanks, fixed it....

File: kdump/include/runtest.sh [hunk 2]
```diff
@@ -339,21 +339,45 @@ ClearReport()
...
-        }
+    rpm -q kexec-tools || {
+        # On Fedora, kexec-tools is not installed by default.
+        # Install kexec-tools and enable kdump service.
+        InstallPackages kexec-tools
+        systemctl enable kdump.service || chkconfig kdump on
+
+        # Back up configurations if kexec-tools is installed for the first time
+        [ -f $"${KDUMP_CONFIG}" -a ! -f "${KDUMP_CONFIG}.bk" ] && cp "${KDUMP_CONFIG}" "${KDUMP_CONFIG}.bk"
+        [ -f $"${KDUMP_SYS_CONFIG}" -a ! -f "${KDUMP_SYS_CONFIG}.bk" ] && cp "${KDUMP_SYS_CONFIG}" "${KDUMP_SYS_CONFIG}.bk"
```
Comment (emanchado) [inline kdump/include/runtest.sh:353]: Oh, haha, good thing I asked, then! I thought it was some obscure syntax thing I didn't know and I wanted to learn it 😂

You forgot to fix these two or...?

Comment (emanchado) [reply kdump/include/runtest.sh:353]: And can you update the branch while at it? Just realised it's out-of-date.

Comment (emmawu) [reply kdump/include/runtest.sh]: Thanks...Hold on, still updating the code. Got some issues on the DNS..
Fingers crossed that there will be no more ${$var} error.

**Comment by emmawu** [comment]
Done. Moved kdump config backup to a func BackupKdumpConfig()...

**Pull request merged**